src/lua/MapApi.cpp

```diff
@@ -1810,8 +1809,7 @@ int LuaContext::map_api_get_entities_in_rectangle(lua_State* l) {
     );
 
     lua_newtable(l);
-    for (auto it = entities.begin(); it != entities.end(); ++it) {
-      EntityPtr entity = *it;
+    for (auto&& entity: entities) {
```

Comment:
I tend to use `auto&&` in range-based `for` loops to deduce the referenceness au `const`ness automagically as well as to handle tricky cases (useful for `std::vector<bool>` for example), but we're indeed not in a template world here...